src/EPR.RegulatorService.Frontend.Web/ViewComponents/RegistrationSubmissions/ProducerPaymentDetailsViewComponent.cs

```diff
@@ -0,0 +1,58 @@
+using System.Threading.Tasks;
+
+using EPR.RegulatorService.Frontend.Core.Extensions;
+using EPR.RegulatorService.Frontend.Core.Models.RegistrationSubmissions;
+using EPR.RegulatorService.Frontend.Core.Services;
+using EPR.RegulatorService.Frontend.Web.ViewModels.RegistrationSubmissions;
+
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.ViewComponents;
+
+namespace EPR.RegulatorService.Frontend.Web.ViewComponents.RegistrationSubmissions;
+
+public class ProducerPaymentDetailsViewComponent(IPaymentFacadeService paymentFacadeService, ILogger<ProducerPaymentDetailsViewComponent> logger) : ViewComponent
+{
+    public async Task<ViewViewComponentResult> InvokeAsync(RegistrationSubmissionDetailsViewModel viewModel)
+    {
+        try
+        {
+            var optionalModel = await paymentFacadeService.GetProducerPaymentDetailsAsync(new ProducerPaymentRequest
+            {
```

Comment:
Request will be updated once we have the details in RegistrationSubmissionDetailsViewModel